01/prvocislo.py

```diff
@@ -0,0 +1,32 @@
+from math import sqrt
+import sys
+
+# one of larger prime numbers is f.e. 10657331232548839
+
+def is_prime_number(num):
+    if num%2 ==0 and num>2 or num==1:				
```

Comment:
```suggestion
    if num % 2 == 0 and num > 2 or num <= 1:
```
Nekladná čísla není nutné ošetřovat mimo tuto funkci. Můžeš prostě vrátit že nejsou prvočísla.